pom.xml

```diff
@@ -85,6 +85,18 @@
         </pluginRepository>
     </pluginRepositories>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>com.oracle.database.jdbc</groupId>
+                <artifactId>ojdbc-bom</artifactId>
```

Comment:
nit: maybe add a comment here asking user to explicitly mention all dependent libs brought in by this bom whenever a version upgrade happens?